cip/CIP2016-06-14-Harmonize-comparability-orderability-equality-and-equivalence.adoc

```diff
@@ -0,0 +1,233 @@
+= CIP2016-06-14 - Harmonize Comparability, Orderability, Equality, and Equivalence
+:numbered:
+:toc:
+:toc-placement: macro
+:source-highlighter: codemirror
+
+*Author:* Mats Rydberg <mats@neotechnology.com>, Stefan Plantikow <stefan.plantikow@neotechnology.com>
+
+[abstract]
+.Abstract
+--
+This CIP intends to introduce and formalise two new concepts, *comparability* and *orderability*, and defines their relationship to the already existing concepts of *equality* and *equivalence*.
```

Comment:
They exist de facto, but I don't know of any specification document. They do need to be defined, I agree.